auth-api/src/auth_api/config.py

```diff
@@ -101,20 +101,29 @@ class _Config:  # pylint: disable=too-few-public-methods
     KEYCLOAK_SERVICE_ACCOUNT_ID = os.getenv('SBC_AUTH_ADMIN_CLIENT_ID')
     KEYCLOAK_SERVICE_ACCOUNT_SECRET = os.getenv('SBC_AUTH_ADMIN_CLIENT_SECRET')
 
+    ENTITY_SVC_CLIENT_ID = os.getenv('ENTITY_SERVICE_ACCOUNT_CLIENT_ID')
```

Comment:
I'm guessing we're using the same entity service account for names as well. Sounds good.